docs/development/setup_development.rst

```diff
@@ -29,12 +29,23 @@ Ubuntu or Debian GNU/Linux
    sudo apt-get update
    sudo apt-get install -y make git
 
+
+Fedora and Other Linux Distributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. note:: To install Docker Engine, you need the 64-bit version of Fedora 30 or higher.
+
+.. code:: sh
+
+   sudo dnf update
```

Comment:
Both this and the original Ubuntu section are missing explanatory text as to why these commands should be run - I mean, it's pretty obvious but a line like "first make sure that git is available on your system" wouldn't hurt imo.

Reply:
I concur with @zenmonkeykstop that adding a bit more context here would be helpful, @joaedwar.

Reply:
I'm not really a Fedora person, but it sounds like the `dnf update` subcommand is a deprecated alias for `dnf upgrade` which is the preferred invocation:

https://dnf.readthedocs.io/en/latest/cli_vs_yum.html#update-and-upgrade-commands-are-the-same